src/layouts/forms/applicationForm.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { NavLink } from "react-router-dom";
+import FormBtn from '../../components/forms/formBtn';
+import FormBox from '../../components/forms/formComponent';
+import './form.css'
+import { Stack } from '@mui/material';
+
+const ApplicationForm = () => {
+    const isAuth = true;
+    return (
+            <div>
+            <form className='wl-st-form'>
+                <div>
```

Comment:
Where is some kind of state or ref in which you will store the values of the form.
Like where is the Name of the person stored once user fills it's name?